vmodules/classes/client.ts

```diff
@@ -15,10 +37,10 @@ module.exports = class Vector extends djs.Client {
     this.version = require(`../../package.json`).version;
 
     this.managers = {
-      assets: require(`../managers/asset_manager.js`),
-      locale: require(`../managers/locale_manager.js`),
-      commands: require(`../managers/command_manager.js`),
-      configs: require(`../managers/config_manager.js`)
+      assets: assetManager,
```

Comment:
technically, it is still possible to use requires in typescript, but i'd rather take advantage of the import/export syntax since it performs better. instead of requiring an entire file, import only requires specific functions.